Makefile

```diff
@@ -26,6 +26,7 @@ RELEASE_DIR ?= release
 DOCS_BUILD_DIR ?= docs/build
```

Comment:
So the goal is to start a webserver that one can connect to in order to check how the docs look like? It would be good to have this explained in the commit log, maybe even put the docker file that you used in the log.

Reply:
Okay, i'll add the explanation, but not the Dockerfile though, i added it in the comment, and ideally we should be putting it in the tree, maybe?

Reply:
Hmm not sure it's worth having it in the tree